conda/recipes/cugraph-nx/meta.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set py_version = environ['CONDA_PY'] %}
+{% set date_string = environ['RAPIDS_DATE_STRING'] %}
+
+package:
+  name: cugraph-nx
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  build:
+      number: {{ GIT_DESCRIBE_NUMBER }}
+      string: py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+
+requirements:
+  host:
+    - python
+  run:
+    - pylibcugraph ={{ version }}
+    - networkx >=3.1
+    - cupy >=12.0.0
+    - python
+
+tests:
+  imports:
+    - cugraph_nx
+
```

Comment:
It's becoming more common to run `pip check` as a sanity check in conda recipes. Dunno if any RAPIDS libraries do this:
```yaml
  commands:
    - pip check
  requires:
    - pip 
```